lib/run-datadog.sh

```diff
@@ -41,6 +40,8 @@ start_datadog() {
     # the conf file requires them to be comma separated only
     # so they must be grabbed separately
     datadog_tags=$(python $DATADOG_DIR/scripts/get_tags.py)
+    export DD_TAGS=""
```

Comment:
```suggestion
    unset DD_TAGS
```
small nit, it seems a little cleaner